staging/src/k8s.io/kms/encryption/aesgcm_test.go

```diff
@@ -0,0 +1,92 @@
+package encryption_test
```

Comment:
```suggestion
package encryption
```

Reply:
Isn't it better to write tests, that are not knowledgeable of the internals?